frontend/src/lib-components/atoms/buttons/AddButton.tsx

```diff
@@ -73,6 +73,7 @@ const StyledButton = styled.button`
 
   &.flipped {
     flex-direction: row-reverse;
+    margin-right: -12px;
```

Comment:
This kind of messes with layout in an unexpected way if the button is used in any different context since it's not even symmetrical. I would just remove the padding from the button, which should align it correctly on every browser.

Reply:
![Screenshot 2021-04-30 at 10 16 39](https://user-images.githubusercontent.com/1533854/116661477-29bace00-a99d-11eb-812c-af7fe5e79045.png)
Hmm yeah, that is an awkward hax, but if I just remove the padding it kind of bothers me that the center of the plus icon and caret are not aligned. But maybe that still looks fine enough? 🤷‍♂️

Reply:
Well that's always going to be a problem with buttons that are not the same size 😄 But at least in figma the icons themselves are not aligned.